browser/ably-prologue.js

```diff
@@ -1,2 +1,12 @@
 (function() {
 	window.Ably = {};
+
+  /*
+    Prevent libraries such as CryptoJS plugging into AMD or CommonJS
+    as the libraries loaded are expected in the `this` context.
+    `require` is only used within the Node.js library, the ably-js browser library
+    is built as a single Javascript file.
+  */
+  var define = undefined,
+      exports = undefined,
+      require = undefined;
```

Comment:
Nope, not incomplete, as mentioned yesterday because we're not using something like Browserify to manage our Browser client library version, we use grunt, see https://github.com/ably/ably-js/blob/karma-test-runner/Gruntfile.js#L131-L146